src/app/public/modules/email-validation/email-validation.directive.ts

```diff
@@ -21,6 +21,12 @@ const SKY_EMAIL_VALIDATION_VALIDATOR = {
 };
 // tslint:enable
 
+/**
+ * Creates an input to validate email addresses. Place this attribute on an `input` element.
+ * If users enter values that are not valid email addresses, an error message appears.
+ * The directive uses `NgModel` to bind data.
+ * @Required
```

Comment:
Does the `@Required` do anything in this situation? I'm pretty sure `@required` is only recognized for properties and parameters...

Reply:
I added the parameter because we tag the directive as required in the live docs:
![image](https://user-images.githubusercontent.com/7850105/91884118-7b1a8e00-ec53-11ea-945f-df4ef5e9dc09.png)

I can remove the parameter here. (I fixed the casing, and it still didn't have any impact.)